Source/WebCore/Modules/mediastream/libwebrtc/LibWebRTCStatsCollector.cpp

```diff
@@ -62,47 +63,48 @@ static inline void fillRtpStreamStats(RTCStatsReport::RtpStreamStats& stats, con
 
     if (rtcStats.ssrc.is_defined())
         stats.ssrc = *rtcStats.ssrc;
+    if (rtcStats.kind.is_defined())
+        stats.kind = fromStdString(*rtcStats.kind);
     if (rtcStats.transport_id.is_defined())
         stats.transportId = fromStdString(*rtcStats.transport_id);
     if (rtcStats.codec_id.is_defined())
         stats.codecId = fromStdString(*rtcStats.codec_id);
-    if (rtcStats.kind.is_defined())
-        stats.kind = fromStdString(*rtcStats.kind);
 }
 
 static inline void fillReceivedRtpStreamStats(RTCStatsReport::ReceivedRtpStreamStats& stats, const webrtc::RTCReceivedRtpStreamStats& rtcStats)
 {
     fillRtpStreamStats(stats, rtcStats);
 
-    // packetsReceived
+    // packetsReceived should be in the base class, but somehow isn't, it's only define for RTCInboundRtpStreamStats
+    // if (rtcStats.packets_received.is_defined())
+    //     stats.packetsReceived = *rtcStats.packets_received;
     if (rtcStats.packets_lost.is_defined())
         stats.packetsLost = *rtcStats.packets_lost;
     if (rtcStats.jitter.is_defined())
         stats.jitter = *rtcStats.jitter;
-    // framesDropped
 }
 
 static inline void fillInboundRtpStreamStats(RTCStatsReport::InboundRtpStreamStats& stats, const webrtc::RTCInboundRtpStreamStats& rtcStats)
 {
     fillReceivedRtpStreamStats(stats, rtcStats);
 
+    // should be in the base class.
+    if (rtcStats.packets_received.is_defined())
+        stats.packetsReceived = *rtcStats.packets_received;
+
     if (rtcStats.track_identifier.is_defined())
         stats.trackIdentifier = fromStdString(*rtcStats.track_identifier);
     if (rtcStats.mid.is_defined())
         stats.mid = fromStdString(*rtcStats.mid);
     if (rtcStats.remote_id.is_defined())
         stats.remoteId = fromStdString(*rtcStats.remote_id);
-
-    if (rtcStats.packets_received.is_defined())
-        stats.packetsReceived = *rtcStats.packets_received;
-    if (rtcStats.frames_dropped.is_defined())
-        stats.framesDropped = *rtcStats.frames_dropped;
-    // full_frames_lost.
-
     if (rtcStats.frames_decoded.is_defined())
         stats.framesDecoded = *rtcStats.frames_decoded;
     if (rtcStats.key_frames_decoded.is_defined())
         stats.keyFramesDecoded = *rtcStats.key_frames_decoded;
+    // Not Implemented
+    // if (rtcStats.frames_rendered.is_defined())
+    //     stats.framesRendered = *rtcStats.frames_rendered;
```

Comment:
I am not sure we want to add these lines, mentioning a comment like // FIXME: Add framesRendered might be good enough.
Ditto below.